src/control/events/pubsub.go

```diff
@@ -102,6 +96,34 @@ func (ps *PubSub) Subscribe(topic RASTypeID, handler Handler) {
 	}
 }
 
+func (ps *PubSub) publish(ctx context.Context, event *RASEvent) {
+	if _, exists := ps.disabledIDs[event.ID]; exists {
+		ps.log.Debugf("event %s ignored by filter", event.ID)
+		return
+	}
+
+	ps.log.Debugf("published @%s: %s", event.Type, event.ID)
```

Comment:
FYI, I have removed these debug statements in one of my PRs. They tend to be spammy when many events are processed. I think these kinds of statements are fine during development, but should be removed before landing.